src/builder/windows.ts

```diff
@@ -5,7 +5,6 @@ import { ERROR_MESSAGES } from '../messages'
 const windows = (ip: string, options?: extendedPingOptions): commandBuilder => {
     let args: Array<string> = [];
     const windowsRootPath = process.env.SystemRoot + '/system32/ping.exe';
-    args.push(ip);
     //NO Allowing Unsanitized user input into spawn.Checking each param and assigning
     if (!options) {
```

Comment:
We should push the ip into the args even if options are not present. Not pushing the args into our command when we have no options is why you have a failing test right now. The other changes look good.

Reply:
Oops, my bad. I did ask what to do when there aren't any args, but I edited that comment back, I thought it would be handled somewhere else. Will change it in the following commit!